jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/WebSocketConstants.java

```diff
@@ -22,14 +22,49 @@
     public static final int SPEC_VERSION = 13;
     public static final String SPEC_VERSION_STRING = Integer.toString(SPEC_VERSION);
 
+    /**
+     * Max Binary Message Size - The maximum size of a binary message which can be received.
+     */
     public static final int DEFAULT_MAX_BINARY_MESSAGE_SIZE = 64 * 1024;
+
+    /**
+     * Max Text Message Size - The maximum size of a text message which can be received.
+     */
     public static final int DEFAULT_MAX_TEXT_MESSAGE_SIZE = 64 * 1024;
+
+    /**
+     * Max Frame Size - The maximum payload size of any WebSocket Frame which can be received.
+     */
     public static final int DEFAULT_MAX_FRAME_SIZE = 64 * 1024;
+
+    /**
+     * Output Buffer Size - The output (write to network layer) buffer size. This is the raw write operation buffer size and has no relationship to the websocket frame.
+     */
```

Comment:
Wrong javadoc? The constant is called _input_ but the javadoc refers to _output_.